src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,43 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.impl.FruitShopServiceImpl;
+import core.basesyntax.service.impl.FruitsHolderServiceImpl;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import core.basesyntax.service.impl.ReportMakerServiceImpl;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import core.basesyntax.strategy.ReturnOperationHandler;
+import core.basesyntax.strategy.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+
 public class HelloWorld {
 
+    private static final String BALANCE = "b";
+    private static final String SUPPLY = "s";
+    private static final String PURCHASE = "p";
+    private static final String RETURN = "r";
+    private static final String INPUT_FILE = "src/main/resources/input.txt";
+    private static final String OUTPUT_FILE = "src/main/resources/output.txt";
+
+    public static void main(String[] args) {
+
+        Map<String, OperationHandler> operationHandlerMap = new HashMap<>();
```

Comment:
Map should contain enum as key 
```suggestion
        Map<FruitTransaction.Operation, OperationHandler> operationHandlerMap = new HashMap<>();
```